Samples~/my-unity-crasher/Scripts/Crasher.cs

```diff
@@ -27,6 +27,13 @@ public void Event_ForceCrash(ForcedCrashCategory category)
 		{
 			Utils.ForceCrash(category);
 		}
+		
+		public void Event_NativeCrashIos()
```

Comment:
Now that the native crash buttons are enabled in the sample do we need this?

Reply:
Those methods are still C# crashes, while this one is native objc crash. 

It is good to have it for checking of ObjectiveC crashes.

Reply:
👍